src/main/java/uk/gov/hmcts/darts/arm/service/impl/ArmResponseFilesProcessorImpl.java

```diff
@@ -93,8 +100,18 @@ public void processResponseFiles() {
         }
     }
 
+    @SuppressWarnings("java:S3655")
+    private void initialisePreloadedObjects() {
+        armDropZoneStatus = objectRecordStatusRepository.findById(ARM_DROP_ZONE.getId()).get();
```

Comment:
Could do this with a post constuct annotation?

Reply:
The automated tasks can run at anytime so we want to get the reference objects each time a new task is run